example/app/scatter.tsx

```diff
@@ -71,6 +79,14 @@ export default function ScatterPage(props: { segment: string }) {
                     strokeWidth={1}
                     color="black"
                   />
+                  {isActive && (
+                    <Text
+                      x={state.x.position}
+                      y={state.y.listenCount.position}
+                      text={state.x.value}
+                      font={font}
+                    />
+                  )}
```

Comment:
This was initially for testing out the `SharedValue<string>` for the update. I forgot to remove this, but maybe we can leave it in? idk, it's not prettied up or anything.